### ISSUE TITLE
Add setting or override to skip decryption

### ISSUE THREAD

**Issue by andrewkrug** [mozilla-iam/cis#41]
For some tests it would be handy not to have to pass an encrypted payload.  Let's have an option that we can set that will skip the decrypt step.

https://github.com/mozilla-iam/cis/blob/master/cis/validation.py#L21

**Comment by andrewkrug** [comment]
Not needed for OKR because we can just use this for testing:

```
def wrapper():
    """Helper for easily calling this from a command line locally.
    Example: `python -c 'import main; main.wrapper()' | jq '.'`

    Only present to facilitate local testing without uploading to
    lambda.  Remove wrapper for production release.  Wrapper() concept
    courtesy of Graham Jones from ThreatResponse.
    """

    event_file = os.path.join(
        os.path.abspath(os.path.dirname(__file__)), 'event.json'
    )

    with open(event_file, 'r') as event_data:
        event = (event_data.read().encode('utf-8'))

    event = encryption.encrypt(event)
    res = handle(event, None)
```

**Comment by johngian** [comment]
Fixed after #45 .
